src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/WritableLaunchProfile.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.VisualStudio.Collections;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug
+{
+
+    /// <summary>
+    /// Represents one launch profile read from the launchSettings file.
+    /// </summary>
+    internal class WritableLaunchProfile : IWritableLaunchProfile
+    {
+        public WritableLaunchProfile()
+        {
+        }
+
+        public WritableLaunchProfile(ILaunchProfile profile)
+        {
+            Name = profile.Name;
+            ExecutablePath = profile.ExecutablePath;
+            CommandName = profile.CommandName;
+            CommandLineArgs = profile.CommandLineArgs; 
+            WorkingDirectory = profile.WorkingDirectory;
+            LaunchBrowser = profile.LaunchBrowser;
+            LaunchUrl = profile.LaunchUrl;
+
+            if(profile.EnvironmentVariables  != null)
+            {
+                EnvironmentVariables = new Dictionary<string, string>(profile.EnvironmentVariables, StringComparer.Ordinal);
+            }
+
+            if(profile.OtherSettings  != null)
+            {
+                OtherSettings = new Dictionary<string, object>(profile.OtherSettings, StringComparer.Ordinal);
+            }
+        }
+
+        /// <summary>
+        /// IWritableDebug profile members
```

Comment:
This isn't really a doc comment, is it? Or if it is, it applies to all the members, not just the top ones.